stack/eval.go

```diff
@@ -44,6 +46,17 @@ func (e *EvalCtx) SetMetadata(sm Metadata) {
 	e.evalctx.SetNamespace("terramate", metaToCtyMap(sm))
 }
 
+// SetEval sets the given environment on the env namespace of the evaluation context.
+// environ must be on the same format as os.Environ().
+func (e *EvalCtx) SetEval(environ []string) {
```

Comment:
SetEval? should not it be `SetEnviron()` or something?

Reply:
The name of the namespace is "eval", so it seemed more straithforward :thinking: (maybe call SetEvalNamespace, but the other namespace setters are just SetNamespaceName).

Reply:
Oh my crap, it should be SetEnv :laughing: ....